04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -314,7 +320,7 @@ george.destroy
 ## Associations
 
 ### Reframing (10 / 165)
-**Question:** We have a lot of choice when it comes to databases, why did we choose to use SQL?
+**Question:** We have a lot of choice when it comes to databases, are we using a SQL db in PostgreSQL?
```

Comment:
Point here is to highlight that SQL is a **relational** database and use that as a segue into Active Record associations. Don't think the PostgreSQL distinction adds much here.

That being said, if this question doesn't sound like it's helpful @adambray, okay with axing it and just diving right into associations.